lib/gateways/pull_request.rb

```diff
@@ -14,7 +14,7 @@ def execute
     end
 
     def approved_pull_requests
-      query = "is:pr user:alphagov state:open author:app/dependabot review:approved"
+      query = "is:pr user:alphagov state:open author:app/dependabot archived:false review:approved"
```

Comment:
Ah - I'm guessing the "Fix tests" commit is because of these changes. Can you squash them together?

Reply:
Done